stack.yaml

```diff
@@ -2,6 +2,7 @@ resolver: lts-6.23
 packages:
   - ./yesod-core
   - ./yesod-static
+  - ./yesod-sibi
```

Comment:
@snoyberg Ah sorry, I was debugging some another work in a different machine and didn't know this branch was a part of the PR. I'm closing the PR and will re-open a new one.